source/guide/3.2/guide-policies.md

```diff
@@ -21,7 +21,7 @@ diamond_package_ref: https://github.com/BrightcoveOS/Diamond
 Policies provide a way of analyzing a stream of events that correspond to a group of nodes (and their instances).
```

Comment:
Let's update the example in this page to use the actual host_failure policy with its properties.
also, let's change the workflow names that is triggered from auto_heal to heal (in the example)